src/Pages/AuditLog/Components/Pagination.js

```diff
@@ -0,0 +1,66 @@
+const Pagination = ({ currentPage, totalPages, goToPage, startIndex, endIndex }) => {
```

Comment:
we already have code for this in
https://github.com/SCE-Development/Clark/blob/1f83f3c3ead67b33a1b53a6858d80106b647b93b/src/Pages/Overview/Overview.js#L185C12-L185C33

maybe just copy paste it into this page? yeah yeah "copy paste is bad" whatever we can be more clever once we have this chain of prs merged